torch_tvm/compiler.cpp

```diff
@@ -169,15 +181,18 @@ tvm::relay::Function TVMCompiler::convertToRelay(
               }
             }
           }
+          if (!skip_user &&
+              tvm_param_tensors_ptr &&
+              std::find(param_indices.begin(),
+              param_indices.end(), input_index) != param_indices.end()) {
+            tvm_param_tensors_ptr->insert_value(input);
```

Comment:
this is much easier to understand with a standard container instead of a custom data structure

Reply:
This is inserting the Value pointer in the cache. Perhaps if `tvm_param_tensor_ptr` is names as `tvm_param_cache_ptr` it might be more readable.